EnglishFilter/sections/specific.txt

```diff
@@ -855,7 +855,8 @@ kotaku.com##.js_movable_ad_slot
 forbes.com##fbs-ad + p > br
 swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
-9tsu.net##.ads-system
+9tsu.net,miomio.us##.ads-system
+9tsu.net,miomio.us##.cactus-sidebar-content > .widget_text
```

Comment:
I don't see any other languages except Japanese. So, please move it to the AdGuard Japanese filter -> specific section.

Reply:
Okay.